practices/documentation.md

```diff
@@ -0,0 +1,70 @@
+---
+slug: documentation
+
+stages:
+  - development
+short_description: Documentation in a software project helps the end users to use the software effectively and saves a lot of time for them while trying to troubleshoot issues.
+
+tags:
+- Documentation
+- Helpful instructions
+- Docusaurus
+- GitBook
+
+keywords:
+- Documentation
+- Helpful instructions
+- Docusaurus
+- GitBook
+
+---
+
+# Documentation
+
+**TL;DR**
+
+Every software should have documentation written for it. It helps users by providing instructions on how to use the software. Follow these guidelines to write good documentation.
+
+## What Is Documentation
+
+Documentation in a software project is a manual that describes the architecture of the software and provides helpful instructions to the users. These instructions include an installation guide, how to use the software, and also guides to resolve common issues users might experience while working with the software.
+
```

Comment:
Please, add an image to your article. It should be one with licence that allows us to use it freely. 
Use similar syntax:
```
![Finger](/files/finger.png)
[Finger](URL source)
```